waffle/utils.py

```diff
@@ -22,7 +22,7 @@ def keyfmt(k, v=None):
         key = prefix + k
     else:
         key = prefix + hashlib.md5((k % v).encode('utf-8')).hexdigest()
-    return key.encode('utf-8')
```

Comment:
Hmm, this was added specifically because unencoded utf8 characters break multiple cache backends (in particular memcache). The Python code is OK with it, but memcached itself was not. If the `CACHE_PREFIX` setting is set to an unsafe string, this will break. `k` should only come from within the library, so that should be OK, and `v` gets encoded and digested, so it should be safe.

At the very least, this would need docs added to the settings reference that made it clear `CACHE_PREFIX` needs to be plain ascii.